python/cuml/linear_model/mbsgd_classifier.pyx

```diff
@@ -172,8 +172,11 @@ class MBSGDClassifier(Base, ClassifierMixin):
             will increase memory used for the method.
         """
         self._set_n_features_in(X)
+        self.cu_mbsgd_classifier._estimator_type = self._estimator_type
```

Comment:
Note that PR #2587 adds automatic detection of solver attributes, so this will not be needed more for long :) 

cc @Nanthini10 after merging this PR, we'll need to address this attribute change there (i.e. remove the assignment)

Reply:
Nice! That's a great improvement